src/providers/DataProvider.js

```diff
@@ -36,6 +42,8 @@ const DataProvider = ({
     });
   }, [consortium?.id]);
 
+  const { data: locationsOfAllTenants } = useLocationsOfAllTenantsQuery();
```

Comment:
it looks like you send 2 equal requests, because `DataProvider` already fetches `tenants` (see `consortiaTenants` in the manifest) and you send it one more time in your `useLocationsOfAllTenantsQuery()` hook